app/components/auth/dummy/lib/function.js

```diff
@@ -0,0 +1,21 @@
+const createDummyUser = () => {
+  return {
+    id: 1,
+    name: "dummy.cat",
+    image:
+      "https://user-images.githubusercontent.com/25859075/29918905-88dcc646-8e5c-11e7-81ec-242bc58dce1b.jpg",
+    email: "dummyuser@rocket.chat",
+    emailVerified: false,
+    phoneNumber: null,
+    displayName: "dummy.cat",
+  };
+};
+
+export const dummyUserLogin = () => {
+  let user = JSON.parse(sessionStorage.getItem("dummy_user"));
```

Comment:
No need for this, you're already checking session storage on `DummyLoginButton`.